pkg/osquery/table/launcher_info.go

```diff
@@ -101,8 +94,8 @@ func generateLauncherInfoTable(configStore types.GetterSetter, LauncherHistorySt
 				"registration_id":     types.DefaultRegistrationID,
 				"identifier":          identifier,
 				"osquery_instance_id": osqueryInstance.InstanceId,
-				"fingerprint":         fingerprint,
-				"public_key":          publicKey,
+				"fingerprint":         "not_in_use",
+				"public_key":          "not_in_use",
```

Comment:
is this okay? We typically don't want to just remove columns from osq tables.

Reply:
Good call on not removing. I'd say NULL or empty string.

Reply:
updated to empty string